### PR TITLE
move everything over to 16 core machines!

### PR DESCRIPTION
**Pull request by pretentious7**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest-16-cores
+    runs-on: ubuntu-latest-16-cores-16-cores
```
Comment (paulgb) [inline .github/workflows/tests.yml:16]: We can't afford 256 cores ;)

**Review by paulgb — COMMENTED** [review]
One typo, rest looks good

File: .github/workflows/tests.yml (hunk 1 above)
Comment (pretentious7) [reply .github/workflows/tests.yml:16]: My sed skills have failed me :laughing:

**Comment by pretentious7** [comment]
O thanks

**Review by paulgb — APPROVED** [review]

**Comment by paulgb** [comment]
Is there any measurable difference here? I wonder if builds are so network IO-bound (especially now that we have caching) that it's a wash?

**Comment by pretentious7** [comment]
for some reason these last builds didn't have anything in the cache.

**Pull request merged**